ceph/rados/serviceability_workflows.py

```diff
@@ -160,47 +161,14 @@ def remove_offline_host(self, host_node_name: str):
 
         # Starting to drain an offline host.
         log.info("Starting to drain an offline host")
-        self.cephadm.shell([f"ceph orch host drain {rm_host.hostname} --force"])
-
-        # Sleeping for 2 seconds for removal to have started
-        time.sleep(2)
-        log.debug(f"Started drain operation on node : {rm_host.hostname}")
-
-        status_cmd = "ceph orch osd rm status -f json"
-        end_time = datetime.datetime.now() + datetime.timedelta(seconds=7200)
-        flag = False
-        while end_time > datetime.datetime.now():
-            out, err = self.cephadm.shell([status_cmd])
-            try:
-                drain_ops = json.loads(out)
-                for entry in drain_ops:
-                    if entry["drain_done_at"] is None or entry["draining"]:
-                        log.debug(
-                            f"Drain operations are going on host {rm_host.hostname} \nOperations: {entry}"
-                        )
-                        raise Exception(
-                            f"drain process for OSD {entry['osd_id']} is still going on"
-                        )
-                    log.info(f"Drain operation completed for OSD {entry['osd_id']}")
-                log.info(f"Drain operations completed on host : {rm_host.hostname}")
-                flag = True
-                break
-            except json.JSONDecodeError:
-                log.info(f"Drain operations completed on host : {rm_host.hostname}")
-                flag = True
-                break
-            except Exception as error:
-                if end_time < datetime.datetime.now():
-                    log.error(f"Hit issue during drain operations: {error}")
-                    raise Exception(error)
-                log.info("Sleeping for 120 seconds and checking again....")
-                time.sleep(120)
-
-        if not flag:
+        if not self.drain_host(host_obj=rm_host):
```

Comment:
Offline hosts cannot be drained. please skip this step and directly proceed to host rm step

Reply:
Yes, this was added by mistake, skipping the drain process in case the host is offline